CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # edge-exchange-plugins
 
+# 0.6.5 (2019-06-04)
```

Comment:
I generally like to see the release notes + version bump in a separate commit whose commit message *is* the version, like "v0.6.5".